department-of-veterans-affairs/caseflow
# Show list of all cases for Veteran on details page

Connects #5743.

Add a list of all cases for the Veteran to the case details page when the "View all cases" link in the case details title bar is clicked.

This PR only attempts to get the mechanics of this feature functional. Styling of this list will be a part of a different PR that will merge back to this branch. This PR will only be merged once the styling in that second PR is complete.

@mdbenjam's refactor of Queue's state will require some changes (that I've called out below), but those should be minor and I think this code is in a reviewable state as is.

### Testing Plan
1. Go to `/queue` as BVASCASPER1.
2. View the case details of any case
3. Click on the "View all cases" link in the case details title bar
4. See a list of cases appear (Rodrigo W O'kon should only have the single case in his list)

| Before | After |
| ------ | ----- |
| ![image](https://user-images.githubusercontent.com/32683958/43732999-3843c730-9981-11e8-997f-29e2e1f0bd6d.png) | ![image](https://user-images.githubusercontent.com/32683958/43732976-21659c64-9981-11e8-8b4c-95cda5a09eea.png) |
